docs/source/dev_guide/algorithms/implicit_als/implicit-alternating-least-squares.rst

```diff
@@ -54,7 +54,7 @@ The ALS model is trained using the implicit ALS algorithm
 by minimizing the following cost function:
 
 .. math::
-	\underset{{x}_{*},{y}_{*}}{\mathrm{min}}\underset{u,i}{\mathrm{\Sigma }}{c}_{ui}{\left({p}_{ui}-{x}_{u}^{T}{y}_{i}\right)}^{2}+\lambda \left(\underset{u}{\mathrm{\Sigma }}{n}_{{x}_{u}}{‖{x}_{u}‖}^{2}+\underset{i}{\mathrm{\Sigma }}{m}_{{y}_{i}}{‖{y}_{i}‖}^{2}\right),
+	\underset{{x}_{*},{y}_{*}}{\mathrm{min}}\underset{u,i}{\mathrm{\Sigma }}{c}_{ui}{\left({p}_{ui}-{x}_{u}^{T}{y}_{i}\right)}^{2}+\lambda \left(\underset{u}{\mathrm{\Sigma }}{n}_{{x}_{u}}{\|{x}_{u}\|}^{2}+\underset{i}{\mathrm{\Sigma }}{m}_{{y}_{i}}{\|{y}_{i}\|}^{2}\right),
```

Comment:
Preview mode shows not correct result. ||xu|| and ||yi|| should be in the formulas.

Reply:
This is the output for this equation, it's the same as the one published:
![image](https://user-images.githubusercontent.com/17865025/96295612-adf7c600-0ff6-11eb-8f89-9767153256f7.png)